tla-io/src/main/scala/at/forsyte/apalache/io/quint/QuintTypeConverter.scala

```diff
@@ -135,13 +135,3 @@ private class QuintTypeConverter extends LazyLogging {
     case QuintUnionT(_, variants) => VariantT1(unionToRowT1(variants))
   }
 }
-
-/**
- * Convert a QuintType to a TlaType1.
- *
- * Constructs a fresh scope for translating Quint type variables to Apalache type variables (see
- * QuintTypeConverter.getVarNo() for details).
- */
-object QuintTypeConverter {
-  def apply(quintType: QuintType): TlaType1 = new QuintTypeConverter().convert(quintType)
```

Comment:
Good catch! I can see how the companion object was creating a new instance every time